app/src/main/java/com/framgia/soundclound/util/designhelper/GridSpacingItem.java

```diff
@@ -0,0 +1,57 @@
+package com.framgia.soundclound.util.designhelper;
+
+
+import android.content.Context;
```

Comment:
ERROR: Unused import: android.content.Context. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports